### ISSUE TITLE
Flutter is not working with JAWS Screenreader

### ISSUE THREAD

**Issue by erhadesso** [flutter/flutter#135649]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

1. Using Text Widget with or without Semantics Wrapper. (see code sample)
2. Running App in Web and open with Chrome
3. Start JAWS Screenreader
4. The widget is not selectable with JAWS screen reader and will not be read
5. The NVDA screen reader has no problems reading the widget 

### Expected results

The Widget Labels should be read and selectable with the arrow keys, just like it works with NVDA.

### Actual results

Compatibility with the JAWS screen reader is very poor to non-existent.

Or am I doing something wrong? Can anyone else share their experiences with Flutter & Jaws?

### Code sample

<details><summary>Code sample</summary>

```dart
                      Semantics(
                        readOnly: true,
                        label: AppLocalizations.of(context).title_login_page_text,
                        child: Text(
                          AppLocalizations.of(context).title_login_page_text,
                          textScaleFactor: Provider.of<FontSizeService>(context).currentFirstHeadlineScaleFactor,
                          style: Theme.of(context).textTheme.headline1,
                        ),
                      ),
```

</details>

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
[√] Flutter (Channel stable, 3.10.6, on Microsoft Windows [Version 10.0.19044.3086], locale de-DE)
    • Flutter version 3.10.6 on channel stable at C:\Users\er\Documents\Software\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f468f3366c (3 months ago), 2023-07-12 15:19:05 -0700
    • Engine revision cdbeda788a
    • Dart version 3.0.6
    • DevTools version 2.23.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[X] Android toolchain - develop for Android devices
    X Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.3086]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 117.0.5938.92
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 117.0.2045.43

[√] Network resources
    • All expected network resources are available.

```

</details>

**Comment by darshankawar** [comment]
@erhadesso 
There's a similar issue open describing your case, https://github.com/flutter/flutter/issues/122607, so please follow-up in it for further updates.
Closing this as a duplicate.
